nyaa/forms.py

```diff
@@ -174,6 +174,9 @@ def validate_category(form, field):
 
 
 class UploadForm(FlaskForm):
+    class Meta:
+        csrf = False
```

Comment:
It's probably something that git or I did by accident. I'll amend it when I can.